package.json

```diff
@@ -50,6 +50,7 @@
   },
   "dependencies": {
     "@types/node": "^10.12.18",
+    "aws-sdk": "^2.656.0",
```

Comment:
This should not be a production dependency.

Reply:
WHHOOOOPS. Fixing now.

Reply:
Fixed.